### PR TITLE
Teryaev/cv sending form

### PR DESCRIPTION
**Pull request by VladTeryaev90**

### CODE REVIEW

File: qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature [hunk 1]
```diff
@@ -0,0 +1,135 @@
+Feature: 1.0 CV_form.feature
+
+Background:
+Given User opens https://career.quantori.com/ in Google Chrome
+And Chooses "Send your CV" form
```
Comment (Starynina) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature]: It seems that some extra description should be added, where to find "Send us your CV" form, as it's not evident that it is in the bottom of the page, for example User should scroll the page down to use "Send us your CV" form

File: qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+When User inputs invalid data <data> in field "Name"
+Then Output <output> should be <Invalid name format>
+And Field "Name" should be highlighted in red
+Examples: 
+| data                               | output              |
+| numbers (1234)                     | Invalid name format |
+| special symbols (&%*)              | Invalid name format |
+| one word without whitespace (Vlad) | Invalid name format |
+| arabian text                       | Invalid name format |
+
```
Comment (Starynina) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature:37]: Good Examples - you first clarify and then provide an abstract example too
Could you write down an example of arabian text? Or is there a problem that it is not displayed in GitHub?

Also I'm not sure if angle brackets are needed in this message: if there is only 1 variant of response. Maybe the quotes are better: "Invalid name format"

File: qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+Scenario Outline: 1.0_05 User inputs invalid data in field "Email"
+When User inputs invalid data <data> in field "Email"
+Then Output <output> should be <Invalid email format>
```
Comment (Starynina) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature:40]: Here is the same about angle brackets, if they are needed in this message or not: <Invalid email format> if there is only 1 variant of response. Maybe the quotes are better: "Invalid email format"

File: qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+When User inputs 111111@domain.com in the field "Email"
+Then Field "Email" should be highlighted in red
+But Field "Email" highlighted in green
+And Error message does not appear
+
+Scenario: 1_0_18 User inputs two "@" in "Email" field
+When User inputs "email@domain@domain.com" in the field "Email"
+Then Message "Invalid email format" should appear under field "Email"
+And Field "Email" should be highlighted in red
+
```
Comment (Starynina) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature]: Seems it's already described in Scenario Outline: 1.0_05 User inputs invalid data in field "Email" and this Scenario can be removed

File: qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+Scenario: 1.0_15 User inputs 499 symbols in the field "Message"
+When User inputs "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Duis aute irure dolor in reprehenderit in voluptate v" in the field "Message"
+Then Field "Message" should be highlighted in green
+And Error message does not appear
+
+Scenario: 1.0_16 User inputs  extra letter in the domain "com" in the field "Email"
+When User inputs "email@domain.comm" in the field "Email"
+Then Field "Email" should be highlighted in red
+But Field "Email" highlighted in green
+And Error message does not appear
```
Comment (Starynina) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature:98]: I like this test! You found a bug!

File: qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+
+Scenario: 1.0_11 User inputs "Name" field with minimum symbols
+When User inputs "C q" in "Name" field
+Then Field "Name" should be highlighted in green
+And Error message does not appear
+
+Scenario: 1.0_12 User inputs only first name in field "Name"
+When User inputs "Vlad" in field "Name" without whitespace
+Then Message "Invalid name format" should appear under field "Name"
+And Field "Name" should be highlighted in red
```
Comment (Starynina) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature]: Seems it's already described in Scenario Outline: 1.0_04 User inputs invalid data in field "Name" and this Scenario can be removed

File: qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+When User inputs "email@domain.comm" in the field "Email"
+Then Field "Email" should be highlighted in red
+But Field "Email" highlighted in green
+And Error message does not appear
+
+Scenario: 1.0_17 User inputs only numbers before domain in "Email" field
+When User inputs 111111@domain.com in the field "Email"
+Then Field "Email" should be highlighted in red
+But Field "Email" highlighted in green
+And Error message does not appear
```
Comment (Starynina) [inline qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature:103]: This scenairio describes the correct behavior, I suppose you can delete BUT and leave a positive test for email field :
"Then Field "Email" should be highlighted in green
And Error message does not appear”

**Review by Starynina — COMMENTED** [review]
Good scenarios, not so complicated and not simple, with correct syntax (only 1 When, Then with “should” structure, etc) and clear headers, test coverage is also sufficient, several test-design techniques are applied, positive and negative tests are presented

File: qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature (hunk 1 above)
Comment (VladTeryaev90) [reply qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature]: Done

File: qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature (hunk 2 above)
Comment (VladTeryaev90) [reply qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature:37]: Thank you Nina. 
Regarding arabian text.. Unfortunatelly i can't  put example( VS 2019 can't save this change)
Also <> or " " i see differents examples from internet when you can use <> or " " for one value

File: qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature (hunk 3 above)
Comment (VladTeryaev90) [reply qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature:40]: Thank you Nina.
Regarding arabian text.. Unfortunatelly i can't put example( VS 2019 can't save this change)
Also <> or " " i see differents examples from internet when you can use <> or " " for one value

File: qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature (hunk 4 above)
Comment (VladTeryaev90) [reply qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature]: Fixed

File: qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature (hunk 5 above)
Comment (VladTeryaev90) [reply qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature:98]: ;)

File: qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature (hunk 6 above)
Comment (VladTeryaev90) [reply qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature]: Fixed

File: qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature (hunk 7 above)
Comment (VladTeryaev90) [reply qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature:103]: Thank you. It's my mistake, i thought that was negative test and  I find a bug) 
 It's good idea I leaved a positive test for email field

File: qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature (hunk 2 above)
Comment (Starynina) [reply qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature:37]: Ok, let's wait for Olga's review

File: qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature (hunk 3 above)
Comment (Starynina) [reply qa/automation/CareerPortal.Tests.Scenarios/3. Teryaev/CV_form.feature:40]: As in previous, let's wait for Olga's review

**Review by Starynina — APPROVED** [review]
